acc_acc_modules/1_annotate_acc_modules.py

```diff
@@ -0,0 +1,373 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:core_acc] *
+#     language: python
+#     name: conda-env-core_acc-py
+# ---
+
+# # Add annotations
+#
+# This notebook takes the dataframe with information about module composition and their labels and adds additional annotations including:
+#
+# 1. Which gene is contained within the modules (both gene id and gene name)
+# 2. Baseline expression and expression in some context of interest
+# 3. How clustered the module is on the genome
+# 4. KEGG pathways that genes are found in
+# 5. GO pathways genes are found in
+# 6. Regulon/operon genes are found in
+#
+# All this information will help _P. aeruginosa_ experiments filter and determine which module might be interesting to explore.
+
+# +
+# %load_ext autoreload
+# %autoreload 2
+# %matplotlib inline
+import os
+import random
+import scipy
+import pandas as pd
+import numpy as np
+from itertools import product
+from core_acc_modules import paths
+
+random.seed(1)
+# -
+
+# User param
+method = "affinity"
```

Comment:
You say "user param", will a future user want to change this? If so, what are the supported options?

Reply:
Very good point I will add text here